package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
     "name": "sarif-viewer",
-    "version": "3.1.2",
+    "version": "3.2.0",
     "lockfileVersion": 2,
     "requires": true,
     "packages": {
         "": {
             "name": "sarif-viewer",
-            "version": "3.1.2",
+            "version": "3.2.0",
```

Comment:
>version": "3.2.0", [](http://example.com/codeflow?start=13&length=18)

Coming in new here, I'm reading this correctly that this is just version tracking for the extension. Correct?

Reply:
Yes, correct.